data/view_model.py

```diff
@@ -0,0 +1,38 @@
+from data.todo_item import Item
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+
+    def todo_items(self):
```

Comment:
Note that even though these don't map to an underlying data field directly, we can still annotate them with `@property` if we choose, and external code can access them as if they were just a field (e.g. with `view_model.todo_items` rather than `view_model.todo_items()`)